examples/sdk-interactive-examples/scripts/createExample.mjs

```diff
@@ -0,0 +1,120 @@
+import fs from 'fs';
+import path from 'path';
+import { execSync } from 'child_process';
+import readline from 'readline';
+
+import { EXAMPLE_CODESANDBOX_PATH_TEMPLATE } from './constants.mjs';
+
+const __dirname = path.dirname(new URL(import.meta.url).pathname);
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+const EXAMPLE_PACKAGE_NAME_PREFIX = '@gooddata/sdk-interactive-';
+const TEMPLATE_DIR = path.join(__dirname, '../', 'examples-template');
+const EXAMPLES_DIR = path.join(__dirname, '../', 'examples');
+
+const RUSH_JSON_PATH = path.join(__dirname, '../../..', 'rush.json');
+const PATH_TO_PRETTIER = path.join(__dirname, '../../..', 'common', 'temp', 'node_modules', '.bin', 'prettier');
+
+let exampleName, exampleTitle, exampleDescription;
+
+function isValidDirectoryName(directoryName) {
+    // Must not contain white spaces or spaces
+    const noWhiteSpaces = !/\s/.test(directoryName);
+
+    // Must start with "example-"
+    const startsWithExample = directoryName.startsWith("example-");
+
+    // Must be a valid directory name
+    // For simplicity, we assume a valid directory name doesn't contain the following characters: \ / : * ? " < > |
+    const isValidChars = !/[\\/:*?"<>|]/.test(directoryName);
+
+    return noWhiteSpaces && startsWithExample && isValidChars;
+}
+rl.question('Enter example directory name (should have prefix "example-"): ', (name) => {
+    if (fs.existsSync(path.join(EXAMPLES_DIR, name))) {
+        console.error('Error: This example directory already exists.');
+        process.exit(1);
+    }
+
+    if(!isValidDirectoryName(name)){
+        console.error('Error: Must not contain white spaces or spaces.');
+        console.error('Error: Must start with "example-"');
+        console.error('Error: Must be a valid directory name does not contain the following characters: \ / : * ? " < > |');
+        process.exit(1);
+    }
+
+    exampleName = name;
+
+    rl.question('Enter example title: ', (title) => {
+        exampleTitle = title;
+
+        rl.question('Enter example description: ', (description) => {
+            exampleDescription = description;
+
+            const NEW_EXAMPLE_DIR = path.join(EXAMPLES_DIR, exampleName);
+
+            // Create new example directory and copy template
+            execSync(`rsync -av --progress ${TEMPLATE_DIR}/ ${NEW_EXAMPLE_DIR}/`);
```

Comment:
Should we mention in the readme, that you need to have installed rsync to work with the examples?

Reply:
I did know that rsync needs to be installed ...